examples/ota-requestor-app/linux/LinuxOTAImageProcessor.h

```diff
@@ -34,6 +35,8 @@ class LinuxOTAImageProcessor : public OTAImageProcessorInterface
     CHIP_ERROR Abort() override;
     CHIP_ERROR ProcessBlock(ByteSpan & block) override;
 
+    void SetOTADownloader(chip::OTADownloader * downloader) { mDownloader = downloader; }
```

Comment:
```suggestion
    void SetOTADownloader(OTADownloader * downloader) { mDownloader = downloader; }
```

I think since it's already in `namespace chip`, this is not needed